src/arch/thread.c

```diff
@@ -11,23 +11,8 @@
  * once all threads are synchronized on the barrier, the function returns
  * true to only one of them.
  *
- * @copyright
```

Comment:
The @copyright tag allows to generate the Doxygen documentation consistently. Should we keep it as is and have SPDX in addition? Should we keep it but have only SPDX? Should we drop this Doxygen tag?
This should apply to all changes in this PR :cry: